### PR TITLE
add 3153 test and update LaunchCluster tests

### PR DESCRIPTION
**Pull request by maryvictol**
This PR provides implementation for LaunchCluster test 3153.
Besides the method for obtaining children run IDs from Nested RunID link has been added to LaunchCluster tests.

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchClusterTest.java [hunk 1]
```diff
@@ -421,6 +442,67 @@ public void hybridAutoScaledClusterCPUDeadlock() {
...
+                .waitForSshLink()
+                .ssh(shell -> shell
+                        .execute("qhost")
+                        .assertOutputContains("HOSTNAME", "global", String.format("%s-%s lx-amd64",
+                                getPipelineName().toLowerCase(), getRunId()))
+                        .sleep(20, SECONDS)
+                        .execute("qstat")
+                        .assertPageContains("sleep", " 1 ")
+                        .assertPageContains("sleep", " qw ")
+                        .assertPageContains("sleep", String.format(" %s ", Integer.parseInt(cpu) + 1))
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchClusterTest.java]: Let's extract to constant, please

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchClusterTest.java [hunk 2]
```diff
@@ -421,6 +442,67 @@ public void hybridAutoScaledClusterCPUDeadlock() {
...
+                .waitForSshLink()
+                .ssh(shell -> shell
+                        .execute("qhost")
+                        .assertOutputContains("HOSTNAME", "global", String.format("%s-%s lx-amd64",
+                                getPipelineName().toLowerCase(), getRunId()), String.format("pipeline-%s",
+                                nestedRunID))
+                        .sleep(20, SECONDS)
+                        .execute("qstat")
+                        .assertPageContains("sleep", " r ")
+                        .assertPageContains("sleep", String.format("main.q@pipeline-%s", nestedRunID))
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchClusterTest.java]: The same as above

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchClusterTest.java (hunk 1 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchClusterTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchClusterTest.java (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchClusterTest.java]: Done

**Review by kamyshova — APPROVED** [review]

**Pull request merged**